gcc/jit/jit-recording.cc

```diff
@@ -31,6 +31,14 @@ along with GCC; see the file COPYING3.  If not see
 #include "jit-recording.h"
 #include "jit-playback.h"
 
+/* This comes from diagnostic.cc.  */
+static const char *const diagnostic_kind_text[] = {
+#define DEFINE_DIAGNOSTIC_KIND(K, T, C) (T),
+#include "diagnostic.def"
+#undef DEFINE_DIAGNOSTIC_KIND
+  "must-not-happen"
+};
+
```

Comment:
I believe the names are now available via `get_diagnostic_kind_text` in `diagnostic.h`

Reply:
Fixed.